app/javascript/components/pages/admin/users-list/utils.js

```diff
@@ -1,27 +1,38 @@
 import { dataToJS } from "../../../../libs";
 import { FILTER_TYPES } from "../../../index-filters";
 
-import { AGENCY, DISABLED } from "./constants";
+import { AGENCY, DISABLED, USER_GROUP } from "./constants";
 
 const searchableAgencies = data => {
   const agencies = dataToJS(data);
 
   return agencies.reduce((acc, agency) => [...acc, { id: agency.id, display_name: agency.name }], []);
 };
 
+const userGroupOptions = data => {
+  const userGroups = dataToJS(data);
+
+  return userGroups
+    ? userGroups.reduce((acc, group) => [...acc, { id: group.unique_id, display_name: group.name }], [])
+    : [];
+};
+
 export const buildUsersQuery = data => {
   return Object.entries(data).reduce((acc, obj) => {
     const [key, value] = obj;
 
     if (key === AGENCY) {
       return { ...acc, [AGENCY]: value?.id };
     }
+    if (key === USER_GROUP) {
```

Comment:
This code is very similar to the one above. Would it be possible to rewrite it to something like: 
```javascript
if ([AGENCY, USER_GROUP].includes(key)){
  return  { ...acc, [key]: value?.id };
}
```

Reply:
Done